docs/data/xrootd/install-storage-element.md

```diff
@@ -72,9 +72,9 @@ These should be replaced with the fully-qualified domain name of the host in que
 You will need to modify the `xrootd-clustered.cfg` on the redirector node and each data node. 
 The following example should serve as a base configuration for clustering. Further customizations are detailed below.
 
-``` file
-all.export %RED%/tmp%ENDCOLOR% stage
-set xrdr = %RED%RDRNODE%ENDCOLOR%
+``` file hl_lines="1 2 11"
+all.export /tmp stage
```

Comment:
Maybe instead of exporting /tmp, we do what we did for the stashcache origin doc: start with a sentence saying "For this installation guide, we assume that the data to be exported is mounted at /mnt/xrootd" and use /mnt/xrootd in the config file.

Reply:
If you don't want to do that or want to do it in a separate PR, that's fine. I'll approve this; feel free to ignore the CI and merge.